console.rb

```diff
@@ -1,17 +1,49 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+
 require_relative "db/connection"
-require_relative "models/apartment"
-require_relative "models/tenant"
+require_relative "models/apartments"
```

Comment:
filenames for classes should be singular! (that's not a technical limitation, but a convention we try to follow as ruby devs)